macOS/DuckDuckGo/WebExtensions/WebExtensionManager.swift

```diff
@@ -34,14 +34,14 @@ protocol WebExtensionManaging {
 
     // Adding and removing extensions
     var webExtensionPaths: [String] { get }
-    func addExtension(path: String)
+    func addExtension(path: String) async
     func removeExtension(path: String)
 
     // Provides the extension name for the extension resource base path
     func extensionName(from path: String) -> String?
 
     // Controller for tabs
-    var controller: WKWebExtensionController? { get }
+    var controller: WKWebExtensionController { get }
```

Comment:
There's no need for the controller to be optional since it can be created at `init` time.